src/containers/flows/MakeABid/MakeABid.tsx

```diff
@@ -107,13 +107,15 @@ const MakeABid = ({ domain, onBid, onClose }: MakeABidProps) => {
 		setStepContent(StepContent.CheckingZAuctionApproval);
 		(async () => {
 			try {
-				const zAuction = await sdk.getZAuctionInstanceForDomain(domain.id);
-				const isApproved = await zAuction.isZAuctionApprovedToTransferNft(
-					account,
-				);
+				const needsApproval =
+					await sdk.zauction.needsToApproveZAuctionToSpendTokens(
+						domain.id,
+						account,
+						BigNumber.from('1000000000'),
```

Comment:
Think this should be changed in the future as it's not uncommon to send billions of a token if the token price is very low. All good for now though as we're just using WILD and Zero.

Reply:
We don't necessarily have to modify the flow Zach said you can just check for a really large approval like a billion because we always approve in the SDK for `Number.MAX_SAFE_INTEGER` anyways